src/SFML/Window/Win32/WindowImplWin32.cpp

```diff
@@ -1011,6 +1023,73 @@ void WindowImplWin32::processEvent(UINT message, WPARAM wParam, LPARAM lParam)
 
             break;
         }
+        case WM_TOUCH:
+        {
+            // Get the number of events
+            std::int16_t num = LOWORD(wParam);
+
+            // Reserve memory
+            std::unique_ptr<TOUCHINPUT[]> events = std::make_unique<TOUCHINPUT[]>(num);
+
+            if (events)
+            {
+                if (GetTouchInputInfo(reinterpret_cast<HTOUCHINPUT>(lParam), num, events.get(), sizeof(TOUCHINPUT)))
```

Comment:
This is wrong, no?

From [the docs](https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-gettouchinputinfo):
> If the function succeeds, the return value is nonzero. If the function fails, the return value is zero. To get extended error information, use the [GetLastError](https://learn.microsoft.com/en-us/windows/desktop/api/errhandlingapi/nf-errhandlingapi-getlasterror) function.

Since the type is not `bool` but `BOOL`, I'd rather use explicit comparison against zero, see my comment above.